tests/js/spec/components/discover/transactionsList.spec.jsx

```diff
@@ -373,4 +373,22 @@ describe('TransactionsList', function () {
       });
     });
   });
+
+  describe('Loading', function () {
+    it('renders loading indicator if forced', async function () {
+      wrapper = mountWithTheme(
+        <TransactionsList
+          api={api}
+          location={location}
+          organization={organization}
+          eventView={eventView}
+          selected={options[0]}
+          options={options}
+          handleDropdownChange={handleDropdownChange}
+          forceLoading
+        />
+      );
+      expect(wrapper.find('LoadingIndicator')).toHaveLength(1);
```

Comment:
Can you also include a prop change in this test to toggle `forceLoading` to false to assert that the list renders as expected?

Reply:
addressed in 675ae7b3b653cd2e2af9164cc3e44dcb79f712c6, the intent was to check that the api call is not made, but i don't think it worked properly anyways.